Submodels/MTFLO_call.py

```
@@ -138,21 +165,23 @@ def LoadForcingField(self,
         self.StdinWrite("")
 
         # Check if file is loaded in successfully.
-        # If error occured, MTFLO will have crashed, so we can check success by checking
-        # if the subprocess is still alive
+        # If error occured, MTFLO will have crashed, so we can check success
+        # by checking if the subprocess is still alive
         if self.process.poll() is not None:
-            raise ImportError(f"Issue with input file tflow.{self.analysis_name}, MTFLO crashed") from None
+            raise ImportError(f"Issue with tflow input file, MTFLO crashed") from None
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)